source/slang/slang-emit-hlsl.cpp

```diff
@@ -93,6 +93,13 @@ void HLSLSourceEmitter::_emitHLSLRegisterSemantic(LayoutResourceKind kind, EmitV
         break;
 
         case LayoutResourceKind::InputAttachmentIndex:
+        {
+            m_writer->emit("[[vk::input_attachment_index("); 
+            m_writer->emit(index);
+            m_writer->emit(")]]");
```

Comment:
This HLSL syntax will be recognized by DXC but not by FXC.
I wonder if there is a way to emit them only for DXC.

Reply:
SubpassInput only works for dxc (fxc does not support sm_6_0+?), I think if this is added to the capability system it would be fine?

Reply:
Why are we generating HLSL from this? This is not supported for d3d, and is added to dxc to produce spirv. We can already produce spirv directly. There is unlikely a need to go from slang to HLSL then to spirv.